### DISCUSSION TITLE
Creating sysroot for Mac OS

### DISCUSSION THREAD

**Discussion by BitShiftedMind** [scasagrande/toolchains_llvm_sysroot#4] [Q&A] [answered]
Thank you for your effort in writing this documentation and providing the accompanying code for building sysroots.

Could you please elaborate on the steps required to create similar sysroots targeting macOS platforms (e.g., `darwin-aarch64`)? My goal is to make bazel builds using `toolchain_llvm` hermetic for darwin targets. If I understand the instructions correctly, the sysroots generated using this method are specifically for Linux builds.

**Comment by scasagrande** [comment, marked as answer]
For macos sysroots you will need a copy from Apple.

For example https://github.com/alexey-lysiuk/macos-sdk

Note that I am not familiar with the licensing terms, if there are any, for using these apple macos sysroots. Proceed as you choose.

**Comment by BitShiftedMind** [comment]
Thank you for providing this additional information.

The reference you linked to looks like a great starting point. I also came across this repository from the Mach Engine project: https://github.com/hexops-graveyard/sdk-macos-13.3. There are some discussions about licensing in their README, and it's also a much smaller distribution (though a little dated).
